### ISSUE TITLE
Importing requests very slow (4 seconds) on Raspberry Pi

### ISSUE THREAD

**Issue by dagwieers** [psf/requests#5080]
We found that importing **requests** on Raspberry Pi is very slow.
It takes 4 seconds.

Given that Kodi calls python addons for every menu change, navigating within addons is dreadfully slow (at least 4 seconds in between clicks).

Profiling and system call tracing output at: https://github.com/Freso/script.module.requests/issues/25

Looking for a way to disable specific packages and keep requests more simple in nature.

## Expected Result
Sub-second behaviour as on Windows and Android.

## Actual Result
It appears that compiling the **rfc3986** regular expressions is taking up most of the meat here.

## Reproduction Steps

```python
#!/usr/bin/python

import sys

sys.path.append('/storage/.kodi/addons/script.module.chardet/lib')
sys.path.append('/storage/.kodi/addons/script.module.urllib3/lib')
sys.path.append('/storage/.kodi/addons/script.module.certifi/lib')
sys.path.append('/storage/.kodi/addons/script.module.idna/lib')
sys.path.append('/storage/.kodi/addons/script.module.requests/lib')

import cProfile

pr = cProfile.Profile()
pr.enable()
import requests

requests.get('https://vrtnu-api.vrt.be/suggest?facets[transcodingStatus]=AVAILABLE').json()
pr.disable()
pr.dump_stats('/tmp/profiling_stats')
```

## System Information

    $ python -m requests.help

*This also takes 5 seconds ;-)*

```
kodi03:~ # python -m requests.help
{
  "chardet": {
    "version": "3.0.4"
  }, 
  "cryptography": {
    "version": ""
  }, 
  "idna": {
    "version": "2.8"
  }, 
  "implementation": {
    "name": "CPython", 
    "version": "2.7.15"
  }, 
  "platform": {
    "release": "4.19.23", 
    "system": "Linux"
  }, 
  "pyOpenSSL": {
    "openssl_version": "", 
    "version": null
  }, 
  "requests": {
    "version": "2.21.0"
  }, 
  "system_ssl": {
    "version": "1000211f"
  }, 
  "urllib3": {
    "version": "1.25.2"
  }, 
  "using_pyopenssl": false
}
```

**Comment by Freso** [comment]
As I said in https://github.com/Freso/script.module.requests/issues/25#issuecomment-489087339, this is almost certainly directly due to https://github.com/urllib3/urllib3/issues/1590 (@dagwieers even mentions `rfc3986` compiling regexes himself as well) and may thus not really be a `requests` issue, though there might be something requests could maybe do to mitigate slowness induced by 3rd party dependencies?

**Comment by nateprewitt** [comment]
This should have been resolved through the outcomes of urllib3/urllib3#1590.
